server/db/schema.sql

```diff
@@ -0,0 +1,23 @@
+-- Run this command in terminal to connect and seed to the database
+-- /psql cars < server/schema.sql
+
+CREATE TABLE IF NOT EXISTS description (
```

Comment:
Looks like you looked into the specific types and tried to pick optimal types. Cool!